docs/guides/scripts.md

```diff
@@ -63,6 +63,14 @@ print("hello world!")
 EOF
 ```
 
+or
+
+```bash
+echo '
+print("hello world!")
+' | uv run -
```

Comment:
Maybe we should just show this as a one-line, e.g. `$ echo 'print("hello world!")' | uv run -` with "console" instead of "bash" syntax? I'd even show this first, then say something like... "Or, if your shell supports heredocs:"

Reply:
That sounds fine to me. My first thought had been to write it as a one-liner, but then I figured that the reason for the way the existing example was written was that most scripts users might want to run would have multiple lines, so the example should make it easy to add more lines to the script. But your suggestion is cleaner, and users can easily convert the one-liner to a multi-liner, so I'll make the change you suggested.

Reply:
👍 thanks!